hera_pspec/uvpspec.py

```diff
@@ -40,8 +40,9 @@ def __init__(self):
         self._cov_model = PSpecParam("cov_model", description=desc, expected_type=str)
         desc = "Power spectrum data dictionary with spw integer as keys and values as complex ndarrays."
         self._data_array = PSpecParam("data_array", description=desc, expected_type=np.complex128, form="(Nblpairts, spw_Ndlys, Npols)")
-        desc = "Power spectrum covariance dictionary with spw integer as keys and values as complex ndarrays. "
-        self._cov_array = PSpecParam("cov_array", description=desc, expected_type=np.complex128, form="(Nblpairts, spw_Ndlys, spw_Ndlys, Npols)")
+        desc = "Power spectrum covariance dictionary with spw integer as keys and values as float ndarrays, stored separately for real and imaginary parts."
```

Comment:
Again, I'd like to see a good rational for doing this.

Reply:
This has been satisfactorily explained. Perhaps consider adding some block comments about why this is.